core/domain/exp_domain.py

```diff
@@ -554,6 +561,18 @@ def validate(self, obj_type, exp_param_specs_dict):
 
         self.outcome.validate()
 
+    def get_fuzzy_rule_spec(self):
+        """Will return the answer group's fuzzy rule, or None if it doesn't
+        exist.
+
+        Returns the first fuzzy rule found in rule_specs because we guarantee
```

Comment:
I'm not sure if this comment is really necessary; the function already implies there is either 1 or 0 fuzzy rule specs in the answer group.